.travis.yml

```diff
@@ -12,7 +11,6 @@ addons:
 cache:
   ccache: true
   directories:
-    - $HOME/.npm
```

Comment:
This is now cached by default. 

> Please note that as of July 2019, npm is cached by default on Travis CI

Reply:
i still prefer the explicit entry; but can you link to where this is quoted from?

Reply:
Sure, here is the source, https://docs.travis-ci.com/user/caching/#npm-cache

Reply:
I do see the value of being explicit, but the cache policy change is the new norm though :)

Reply:
That says
> This caches $HOME/.npm or node_modules, depending on the repository’s structure

which makes me more strongly prefer the explicit form.